modules/ROOT/pages/using.adoc

```diff
@@ -23,6 +23,66 @@ Only one preset can be active at a time.
 * xref:installing.adoc#minimum-system-requirements[Minimum system requirements].
 * xref:configuring.adoc#changing-the-selected-preset[Changing the selected preset].
 
+[id='about-pullsecrets']
+== About Pull Secrets
+While starting cluster configured with {openshift} or {ushift} preset, {prod} would request you to provide a pull secret. It's required to pull some {ocp} specific container images.
```

Comment:
```suggestion
While starting cluster configured with {openshift} or {ushift} preset, {prod} would request you to provide a pull secret. It's required to pull container images from Red Hat container registries.
```

Reply:
The sentence itself is confusing, too long and the "While"-statement is not correctly used.